warn/warn_docstring.go

```diff
@@ -12,7 +12,7 @@ import (
 // a public function is required to have a docstring.
 const FunctionLengthDocstringThreshold = 5
 
-// getDocstrings returns a docstring of the statemenets and true if it exists.
+// getDocstrings returns a docstring of the statements and true if it exists.
 // Otherwise it returns the first non-comment statement and false.
 func getDocstring(stmts []build.Expr) (build.Expr, bool) {
```

Comment:
If you don't need it, don't return the first statement anymore.

Reply:
It's technically still used (no docstring warnings are not shown if the file has no non-comment statements), but I don think that makes much sense, will remove.

Reply:
I'll remove this logic, but the first returned argument is still used when I need to analyze the docstring contents.